### ISSUE TITLE
How to remove nfs?

### ISSUE THREAD

**Issue by frankcarey** [adlogix/docker-machine-nfs#34]
So, if we want to uninstall the nfs support, is there an easy way to do that?

**Comment by brandontamm** [comment]
@tonivdv - I'm guessing, but I think just delete docker-machine-nfs from /usr/local/bin

**Comment by frankcarey** [comment]
no, sorry.. I mean to remove the nfs mount that docker-machine-nfs creates (basically a `docker-machine-nfs undo`)

**Comment by tonivdv** [comment]
Hey @frankcarey ,

Seems a nice feature to add :) . But basically if you remove the /var/lib/boot2docker/bootlocal.sh from your box and restart it should return to the basic functionality. Additionally you can clean the /etc/exports too by removing the line related to your box.

**Comment by kadishmal** [comment]
I need this feature, too. After completing the project which required this NFS volume support for local development, I need to switch to another project which can't use NFS volumes, so it's necessary to remove the NFS support.

I followed @tonivdv's suggestion and commented out the contents of the `/var/lib/boot2docker/bootlocal.sh` file, just in case one day I need NFS back. Would be nice to have a command that would do this cleanup.
